ci-operator/step-registry/ipi/conf/vsphere/template/ipi-conf-vsphere-template-commands.sh

```diff
@@ -42,17 +38,27 @@ else
   vm_template="${RHCOS_VM_TEMPLATE}"
 fi
 
-cat >> "${SHARED_DIR}"/enable_template_content.txt << EOF
+cat > "${PATCH}" << EOF
+platform:
+  vsphere:
     failureDomains:
     - name: generated-failure-domain
-      region: generated-region
-      server: "${vsphere_url}"
       topology:
-        computeCluster: /${vsphere_datacenter}/host/${vsphere_cluster}
-        datacenter: ${vsphere_datacenter}
-        datastore: /${vsphere_datacenter}/datastore/${vsphere_datastore}
         template: /${vsphere_datacenter}/vm/${vm_template}
-        networks:
-        - ${vsphere_portgroup}
-      zone: generated-zone
 EOF
+yq-go m -x -i "${CONFIG}" "${PATCH}"
+
+#cat >> "${SHARED_DIR}"/enable_template_content.txt << EOF
```

Comment:
I think we can drop the comment lines now.

Reply:
ok